Description: Correct man file mdoc syntax
 Two '.Bl' directives were present without the corresponding '.El' directives
 to close the lists created therein. This patch corrects that deficiency and
 corrects other minor issues such as misspellings.
Author: Peter J. Mello <admin@petermello.net>
Origin: other
Forwarded: no
Last-Update: 2020-05-13

--- hardcode-tray-4.3.orig/data/man/hardcode-tray.1
+++ hardcode-tray-4.3/data/man/hardcode-tray.1
@@ -47,17 +47,19 @@ to use a certain icon size, where SIZE is one of
 .It --version
 Prints the version of
 .Nm
+.El
 .Sh FILES
 .Nm
-uses a JSON configuration file at \fI~/.config/hardcode-tray.json\fR, is it exists. The file supports the following options:
+uses a JSON configuration file at \fI~/.config/hardcode-tray.json\fR, if it exists. The file supports the following options:
 .Bl -tag -width Ds
 .It blacklist
 List applications whose tray icons will never be patched.
 .It conversion_tool
-Default tool to be used. Supersedes autodetection.
+Default tool to be used; supersedes auto-detection.
 .It icons
 Object containing the \fItheme\fR and \fIsize\fR of the icon to be used.
 .It backup_ignore
 Boolean setting whether to ignore backing up the original icon.
 .It scaling_factor
-Widgets scaling factor, autodetected in GNOME, KDE and Cinnamon.
+Widgets scaling factor, auto-detected in GNOME, KDE and Cinnamon.
+.El
